src/utiltime.cpp

```diff
@@ -40,6 +40,14 @@ int64_t GetTimeMicros()
             boost::posix_time::ptime(boost::gregorian::date(1970,1,1))).total_microseconds();
 }
 
+/** Return a time useful for the debug log */
+int64_t GetLogTimeMicros()
+{
+    if (nMockTime) return nMockTime*1000000;
```

Comment:
Yeah I have some code I use to test that plays back historical data to a node and uses mocktime, and having the debug log have (mostly) consistent timestamps is helpful to me, so I figured I'd preserve the existing behavior.